Software/Python/test_ir_rec.py

```diff
@@ -0,0 +1,12 @@
+# Test IR Receiver
```

Comment:
@CleoQc this is a quick python test program I put in there to test the IR Receiver.  You should see numbers flash across the screen if you're running it, have an IR receiver in D11, and are pressing buttons on your remote control.